.github/config/linux.json

```diff
@@ -6,14 +6,6 @@
     ],
     "targets": [
  	"base_unittests",
-        "sql_unittests",
```

Comment:
All these targets fail with the existing changes. Enabling use_partition_alloc and restoring the dependency of //base on partition_alloc:partition_alloc is not trivial .  I can work on this after setting up the evergreen toolchain / someone else can work on this from the base migration team . wdyt @andrewsavage1 , @kaidokert